lib/openapi/proofs.ts

```diff
@@ -33,6 +33,111 @@ const commonResponses = {
 }
 
 export const proofsPaths: ZodOpenApiPathsObject = {
+  "/proofs": {
+    get: {
+      tags: ["Proofs"],
+      summary: "List proofs",
+      description: "Retrieve a filtered and paginated list of proofs.",
+      parameters: [
+        {
+          name: "team",
+          in: "query",
+          description: "Filter by team slug",
+          required: false,
+          schema: {
+            type: "string",
+          },
+        },
+        {
+          name: "block",
+          in: "query",
+          description: "Filter by specific block number",
+          required: false,
+          schema: {
+            type: "number",
+          },
+        },
+        {
+          name: "limit",
+          in: "query",
+          description: "Number of proofs to return (default: 100, max: 1000)",
+          required: false,
+          schema: {
+            type: "integer",
+            default: 100,
+            minimum: 1,
+            maximum: 1000,
+          },
+        },
+        {
+          name: "offset",
+          in: "query",
+          description: "Number of proofs to skip for pagination (default: 0)",
+          required: false,
+          schema: {
+            type: "integer",
+            default: 0,
+            minimum: 0,
+          },
+        },
+      ],
+      responses: {
+        "200": {
+          description: "List of proofs retrieved successfully",
+          content: {
+            "application/json": {
+              schema: z.object({
+                proofs: z.array(
```

Comment:
Testing this locally, I think maybe this schema has changed bc it's not being transformed? It would return the entire proof, correct?

Reply:
@yourbuddyconner this is still outdated, correct? I believe if this is updated we can merge this.